main.js

```diff
@@ -37,37 +70,52 @@ function doWork() {
     document.querySelectorAll(globalSelectors.viewCount)
   );
 
-  const articleViewDateSection = document.querySelector(globalSelectors.articleDate);
+  const articleViewDateSection = document.querySelector(
+    globalSelectors.articleDate
+  );
 
-  if(articleViewDateSection) {
-    let rootDateViewsSection = articleViewDateSection.parentElement.parentElement.parentElement;
+  if (articleViewDateSection) {
+    let rootDateViewsSection =
+      articleViewDateSection.parentElement.parentElement.parentElement;
 
-    if(rootDateViewsSection?.children.length === 1) {
+    if (rootDateViewsSection?.children.length === 1) {
       // we're dealing with the <time> element on a quote retweet
       // do globalSelector query again but with 2nd result
-      rootDateViewsSection = document.querySelectorAll(globalSelectors.articleDate)[1].parentElement.parentElement.parentElement;
+      rootDateViewsSection = document.querySelectorAll(
+        globalSelectors.articleDate
+      )[1].parentElement.parentElement.parentElement;
     }
 
     // if there are more than 4, we already added the paycheck value
-    if(rootDateViewsSection?.children.length < 4) {
-
+    if (rootDateViewsSection?.children.length < 4) {
       // clone 2nd and 3rd child of rootDateViewsSection
-      const clonedDateViewSeparator = rootDateViewsSection?.children[1].cloneNode(true);
+      const clonedDateViewSeparator =
+        rootDateViewsSection?.children[1].cloneNode(true);
       const clonedDateView = rootDateViewsSection?.children[2].cloneNode(true);
 
       // insert clonedDateViews and clonedDateViewsTwo after the 3rd child we just cloned
-      rootDateViewsSection?.insertBefore(clonedDateViewSeparator, rootDateViewsSection?.children[2].nextSibling);
-      rootDateViewsSection?.insertBefore(clonedDateView, rootDateViewsSection?.children[3].nextSibling);
+      rootDateViewsSection?.insertBefore(
+        clonedDateViewSeparator,
+        rootDateViewsSection?.children[2].nextSibling
+      );
+      rootDateViewsSection?.insertBefore(
+        clonedDateView,
+        rootDateViewsSection?.children[3].nextSibling
+      );
 
       // get view count value from 'clonedDateViewsTwo'
-      const viewCountValue = clonedDateView?.querySelector(innerSelectors.articleViewAmount)?.textContent;
+      const viewCountValue = clonedDateView?.querySelector(
+        innerSelectors.articleViewAmount
+      )?.textContent;
       const dollarAmount = convertToDollars(viewCountValue);
 
       // replace textContent in cloned clonedDateViews (now 4th child) with converted view count value
-      clonedDateView.querySelector(innerSelectors.articleViewAmount).textContent = "$" + dollarAmount;
+      clonedDateView.querySelector(
+        innerSelectors.articleViewAmount
+      ).textContent = "$" + dollarAmount;
```

Comment:
Shouldn't we also stringify `dollarAmount` based on the locale? Or does it do that automatically? If not, I can make a simple PR for this.